msys2-runtime/update-patches.sh

```diff
@@ -19,8 +19,11 @@ die "Clean worktree required"
 git rm 0*.patch ||
 die "Could not remove previous patches"
 
+merging_rebase_start="$(git -C src/msys2-runtime \
+    rev-parse --verify --quiet ^HEAD^{/Start.the.merging.rebase})"
```

Comment:
If this does find something, `$merging_rebase_start` would be `^<commit hash>`. If it doesn't, it's empty. Tested locally with a different expression.